Pipfile

```diff
@@ -0,0 +1,22 @@
+[[source]]
```

Comment:
is this file mandatory for tidelift?

Reply:
@auvipy If you want to provide a lock file, then either Pipfile.lock or poetry.lock according to https://support.tidelift.com/hc/en-us/articles/4406293161492-Compatible-languages-and-package-files

Reply:
OK I am familiar with both, but not so with tidelift CLI :D I was able to make tidelift authenticate work though

Reply:
but should I put key in a secret env variable in the local tidelift project?